guides/common/modules/con_using-pxe-to-provision-hosts.adoc

```diff
@@ -51,3 +51,102 @@ endif::[]
 For BIOS provisioning, you must associate a PXELinux template with the operating system.
 For UEFI provisioning, you must associate a PXEGrub2 template with the operating system.
 If you associate both PXELinux and PXEGrub2 templates, {Project} can deploy configuration files for both on a TFTP server, so that you can switch between PXE loaders easily.
+
+.SecureBoot Support
+
+SecureBoot expects to follow a chain of trust from the start of the host to the loading of Linux kernel modules.
+The very first shim that is loaded basically determines which distribution is allowed to be booted or kexec’ed until next reboot.
+
+The following assumes host systems with enabled SecureBoot (User Mode) and default platform certificates.
+If you are using custom keys, you have to sign the corresponding components accordingly.
+
+To make this POC working you need to provide all shim and GRUB2 binaries manually under `/usr/local/share/bootloader-universe/<os>/`.
+You need to set SELinux to permissive when testing.
+
+Create the following directory structure on the server according to your needs and supported operating systems:
+[options="nowrap" subs="+quotes"]
+----
+[root@or ~]# install -o foreman-proxy -g foreman-proxy -d /usr/local/share/bootloader-universe/{almalinux,rocky_linux,ubuntu}/
```

Comment:
```suggestion
# install -o foreman-proxy -g foreman-proxy -d /usr/local/share/bootloader-universe/{almalinux,rocky_linux,ubuntu}/
```

I'd also go with `_My_Operating_Systems_` or mark this as example.

Where does "almalinux" come from? Is alma-linux or "rocky-linux" als correct? Does this have to match the OS name from facter?

Reply:
As all commands will be executed on the same machine (=Foreman server) it's OK to strip the prompt.

> Where does "almalinux" come from? Is alma-linux or "rocky-linux" als correct? Does this have to match the OS name from facter?

https://github.com/theforeman/foreman-documentation/pull/2145/files#diff-2dae64f772bb99ae82fd42450722362d1d035b858664cdbc26ce5d0c9df881c5R71-R78